### ISSUE TITLE
Regarding the names of name origin groups, what do you think about these new names:

### ISSUE THREAD

**Issue by dhimmel** [greenelab/iscb-diversity-manuscript#94]
Regarding the names of name origin groups, what do you think about these new names:

- Celtic/English names
- European names
- East Asian names
- Hispanic names
- South Asian names
- Arabic names
- Hebrew names
- African names
- Greek names
- Nordic names

@cgreene @dhimmel @arielah

_Originally posted by @trang1618 in https://github.com/greenelab/iscb-diversity-manuscript/issues/56#issuecomment-602654415_

**Comment by dhimmel** [comment]
So are we still grouping Greek with European even though Greek is its own top-level nameprism group?

**Comment by cgreene** [comment]
I still think lettered groupings are a better way to go. When you consider how far removed we are from the actual naming history of each of the names that comprise the training set, it's hard to say we've actually made categories according to this taxonomy. I feel less strongly about that than I feel about using the top level nameprism group, which I do think we should do. If we don't have enough training examples for robust performance on some we will have to just drop the categories.

**Comment by trangdata** [comment]
> So are we still grouping Greek with European even though Greek is its own top-level nameprism group?

My bad. I meant to add Greek names there. Edited.

**Comment by trangdata** [comment]
>  I feel less strongly about that than I feel about using the top level nameprism group, which I do think we should do. If we don't have enough training examples for robust performance on some we will have to just drop the categories.

Performance seems [robust](https://greenelab.github.io/iscb-diversity/21.draw-roc.html#auroc). Prediction of African names has the lowest auROC (92.2%) likely because there are fewer Wikipedia entries.

> I still think lettered groupings are a better way to go. When you consider how far removed we are from the actual naming history of each of the names that comprise the training set, it's hard to say we've actually made categories according to this taxonomy.

I understand why we would want to go with lettered groupings. However, with 10 categories, I think it's very challenging to map the letters/colors to the region. We did our best in terms of removing the mix of religions/country/area in the NamePrism group names and focus on the language. We will certainly point out the potential mismatch (e.g., as Iddo pointed out in #27, name of a person from Israel can be of Hebrew, Arabic or European origin - which is reflected in the heatmap), but I do think the group names helped the reader see how we made our predictions.
